setup.sh

```diff
@@ -361,7 +361,7 @@ elif [[ ${versionArg} == true ]]; then
 	version="${versionVal}"
 	case ${version} in
 	"1.21.4")
```

Comment:
I am very sorry about the messy code structure, but this is the 6th place which needs adjustment like the others. `1.21.4` => `1.21.5`

Reply:
Yeah maybe, although they span multiple files and would need to be in a common file which is read by every script like `server.settings`, maybe I will try something someday, maybe not.